packages/manager/tools/component-rollup-config/src/plugins/translation-ui-router.ts

```diff
@@ -20,14 +20,16 @@ const removeProperty = (code, magicString, start, end) => {
   }
```

Comment:
Updated on `develop` but target branch `feat/web-ca` needs to be rebased. Only purpose of updating it in  this PR is to be able to check that branch builds and that behaviour is correct on a prod environment